src/web/components/ClickToView.tsx

```diff
@@ -14,20 +14,21 @@ type Props = {
 	onAccept?: Function;
 	isTracking: boolean;
 	source?: string;
-	sourceDomain: string;
+	sourceDomain?: string;
+	abTests: CAPIType['config']['abTests'];
 };
 
 const roleTextSize = (role: RoleType) => {
 	switch (role) {
 		case 'immersive':
 		case 'inline':
 		case 'showcase': {
-			return textSans.small();
+			return textSans.medium();
 		}
 		case 'halfWidth':
 		case 'supporting':
 		case 'thumbnail': {
-			return textSans.xsmall();
+			return textSans.small();
```

Comment:
Is it worth making these design changes separately? I'd be interested to check how these new font sizes render with different roles and with different sizes

Reply:
So these changes actually implement the original designs, there was some inline styling in the story books which was interfering with the styles from the component. 

Ive reworked the storybooks to fix that problem so you should be able to see the storybooks for the different roles as they would appear in rendered articles.

Without these changes the overlay looks bad :)